.github/workflows/build.yml

```diff
@@ -20,34 +20,25 @@
 
 name: build-and-analyse
 
-on: [push]
+on: [ push ]
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        java-version: [11]
+        java-version: [ 17 ]
```

Comment:
Given how AEM projects are basically stuck with Java 11 (and sometimes 8), let's make sure this gets tested with Java 11 as well. I think some folks still use the plugin with Java 8, but that's just with the Sonar Scanner, rather than the server, see https://github.com/wttech/AEM-Rules-for-SonarQube/issues/201 but this action is responsible for the testing of SonarQube server, which doesn't support Java 8 anwyay.